lib/spack/llnl/util/filesystem.py

```diff
@@ -861,17 +961,6 @@ def is_exe(path):
     return os.path.isfile(path) and os.access(path, os.X_OK)
 
 
-@system_path_filter
-def get_filetype(path_name):
-    """
-    Return the output of file path_name as a string to identify file type.
-    """
-    file = Executable("file")
-    file.add_default_env("LC_ALL", "C")
-    output = file("-b", "-h", "%s" % path_name, output=str, error=str)
-    return output.strip()
```

Comment:
(question) I assume this is removed because nothing calls it?

Reply:
Correct. That said, the investigation consisted of me grep-ing for uses and then running the unit tests to see if anything broke.